client/src/login/LoginForm.js

```diff
@@ -0,0 +1,25 @@
+import React, { PropTypes } from 'react';
+import { Field, reduxForm } from 'redux-form';
+
+export const LoginForm = reduxForm({
```

Comment:
(also, frustratingly, an error that only seems to be cleared by restarting npm start)